frappe/public/js/frappe/form/toolbar.js

```diff
@@ -30,7 +30,8 @@ frappe.ui.form.Toolbar = Class.extend({
 	},
 	set_title: function() {
 		if(this.frm.meta.title_field) {
-			var title = strip_html((this.frm.doc[this.frm.meta.title_field] || "").trim() || this.frm.docname);
+			let title_field = (this.frm.doc[this.frm.meta.title_field].toString() || "").trim();
```

Comment:
`isString()` is not the right function for this case.

`number.toString()` will fail.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/toString

Use `cstr()` instead